app/src/main/res/layout/fragment_skill_details.xml

```diff
@@ -376,6 +376,64 @@
                 android:layout_marginTop="@dimen/margin_small"
                 android:visibility="gone" />
 
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="16dp"
+                android:text="@string/feedback"
+                android:textColor="@color/md_black_1000"
+                android:textSize="@dimen/text_size" />
+
+            <TextView
+                android:id="@+id/tv_post_feedback_desc"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/desc_post_feedback"
+                android:visibility="gone" />
+
+            <LinearLayout
+                android:id="@+id/layout_post_feedback"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="horizontal"
+                android:visibility="gone">
+
+                <android.support.design.widget.TextInputLayout
+                    android:id="@+id/feedback"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_weight="1">
+
+                    <android.support.design.widget.TextInputEditText
+                        android:id="@+id/et_feedback"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:hint="@string/hint_feedback"
+                        android:inputType="text" />
+
+                </android.support.design.widget.TextInputLayout>
+
+                <Button
+                    android:id="@+id/button_post"
+                    android:layout_width="@dimen/button_width_skill_details"
+                    android:layout_height="@dimen/button_height_skill_details"
+                    android:layout_gravity="bottom"
+                    android:layout_marginBottom="@dimen/margin_small"
+                    android:background="@color/md_cyan_300"
```

Comment:
Never use background on buttons

Reply:
@iamareebjamal But I need to change the color of this button. Should I use 'style' instead?

Reply:
@iamareebjamal Does this lead to any problems with different devices?

I checked the official Android documentation. It says the following : 

<img width="1440" alt="screen shot 2018-06-26 at 1 27 26 am" src="https://user-images.githubusercontent.com/30979369/41872763-88281b78-78e0-11e8-98c6-98f2d1c234be.png">